bundle/project/build.properties

```diff
@@ -0,0 +1 @@
+sbt.version=1.6.2
```

Comment:
This would seem to clash with the definition in `project/build.properties`.  Is it used by anything?  I'm skeptical that the build will work with a newer version -- it's kind of complicated by OSGi.